btree/btree.c

```diff
@@ -5214,6 +5308,8 @@ struct btree_ut_tree_info {
 	struct m0_btree *tree;
 };
 
+#define MAX_RECS_PER_THREAD  100000 /** Records per thread will work with */
```

Comment:
Please define `MAX_RECS_PER_THREAD` and `btree_ut_tree_info` at one place where other UT related macros/functions are declared.

Reply:
I have kept them here since they are used only by the functions defined below this line. If I move them up then there might be some confusion if we were to look just at the define's or structures.

Reply:
I have moved the defines in an enum and removed the struct btree_ut_tree_info as it is not needed.